docs/toc.html

```diff
@@ -27,7 +27,8 @@
                 <li><a href="#quickstart">1.3 Quick Start</a>
                 <li><a href="#ecosystem">1.4 Ecosystem</a>
                 <li><a href="#upgrade">1.5 Upgrading</a>
-                <li><a href="#docker">1.6 Docker</a>
+                <li><a href="/{{version}}/zk2kraft">1.6 Differences Between KRaft mode and ZooKeeper mode</a>
```

Comment:
In https://kafka.apache.org/40/documentation/, the link from the index page on the left points to the beginning of the doc page, not the zk2kraft page.

Also, in the main doc page, we still have `1.6 Docker`.

Unrelated to this PR, the following has a missing `)`.  Perhaps we could fix them together.

 `Apache Kafka 4.0 ships with a brand-new group coordinator implementation (See here.`

Reply:
we will file a minor later